data.sql

```diff
@@ -3,3 +3,33 @@
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
+
+/* day 1 database */
+
+/* inserting values in animals table */
+
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Agumon', '2020-02-03', 0, true, 10.23);
+
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Gabumon', '2018-11-15', 2, true, 08.00);
+
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Pikachu', '2021-01-07', 1, false, 15.04);
+
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Davimon', '2017-05-12', 5, true, 11.00);
+
+/* day 2  database */
+
+/*  His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.  */
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ( 'Charmander', '2020-02-08', 0, false, 11.00);
+/* Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times. */
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ( 'Plantmon', '2021-11-15', 2, true, -5.07);
+/* His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times. */
```

Comment:
- [ ] It would be nice if you add the correct weight for Plantmon because you are adding 5.07 and the correct weight is 5.7 🤔 Kindly add the correct weight to follow the project requirements 😊